src/seedu/addressbook/ui/TextUi.java

```diff
@@ -12,29 +11,124 @@
 import java.util.Optional;
 import java.util.Scanner;
 
-/**
- * Text UI of the application.
- */
-public class TextUi {
 
-    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+class Formatter {
+	
+	/** A decorative prefix added to the beginning of lines printed by AddressBook */
     private static final String LINE_PREFIX = "|| ";
 
     /** A platform independent line separator. */
     private static final String LS = System.lineSeparator();
 
     private static final String DIVIDER = "===================================================";
-
+    
     /** Format of indexed list item */
     private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
 
 
     /** Offset required to convert between 1-indexing and 0-indexing.  */
     public static final int DISPLAYED_INDEX_OFFSET = 1;
 
+    /**
+     * @return formatted prompt msg.
+     */
+	public String getUserCommandPrompt() {
+		
+		return LINE_PREFIX + "Enter command: ";
+		
+	}
+	
+	/**
+     * @param rawInputLine full raw user input line.
+     * @return formatted command line
+     */
+	public String getFullUserCommand(String fullCommand){
+		
+		return "[Command entered:" + fullCommand + "]";
+	}
+
+	/**
+     * @param msg to be shown
+     * @return formatted msg 
+     */
+	public String showToUser(String... message) {
```

Comment:
might be better to change the name to something else rather than `show` because this method simply formats and not print